src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/MySqlTranslator.java

```diff
@@ -275,6 +276,11 @@ public String translate(DbColumn c) {
         }
     }
 
+    @Override
+    public String translate(final With with) {
+        throw new OperationNotSupportedRuntimeException("MySQL 5.7 does not support WITH");
```

Comment:
Please omit the version, you cannot know which version is being used.  
Although wrong, lets just assume that all MySQL versions do not support this.

Reply:
Ok